corehq/apps/cloudcare/static/cloudcare/js/form_entry/spec/entries_spec.js

```diff
@@ -100,6 +101,31 @@ hqDefine("cloudcare/js/form_entry/spec/entries_spec", function () {
             assert.isTrue(spy.calledTwice);
         });
 
+        it('Should return MultiDropdownEntry', function () {
+            var entry;
+
+            questionJSON.datatype = constants.MULTI_SELECT;
+            questionJSON.style = { raw: constants.MINIMAL};
+            questionJSON.choices = ['a', 'b'];
+            questionJSON.answer = [1, 2]; // answer is based on a 1 indexed index of the choices
+
+            entry = formUI.Question(questionJSON).entry;
+            assert.isTrue(entry instanceof entries.MultiDropdownEntry);
+            assert.equal(entry.templateType, 'multidropdown');
+            assert.equal(entry.placeholderText, 'Please choose an item')
+
+            assert.isTrue(entry instanceof entries.MultiSelectEntry);
+            assert.sameMembers(entry.answer(), [1, 2]);
+            assert.sameMembers(entry.rawAnswer(), ['a', 'b']);
+
+            entry.answer([1]);
+            entry.choices(['a', 'c']);
+            this.clock.tick(1000);
+            assert.equal(spy.calledOnce, true);
+            assert.equal(entry.rawAnswer()[0], 'a');
+            assert.equal(entry.answer()[0], 1);
+        });
```

Comment:
>seems to me like most of the behavior it tests is already accomplished in the other MultiSelectEntry test

I see what you mean. My argument in favor of this test is that it doesn't make tests take any longer to run and it tests a template type that wasn't being tested. I definitely like checking the default `placeholderText` (more on that below).

>Seems like you could just add the line assert.equal(entry.placeholderText...

My reasoning for checking `placeholderText` here and in `DropdownEntry` is that those two already had default placeholder text. I wanted to make sure the new placeholder logic I added to `Entry` didn't interfere with the default for either. 

>it might make more sense to test a freetext input field...

I'm assuming delivery will want to customize other input placeholders, so this allows for that inevitability. I agree that having a test on `FreeTextEntry` with custom placeholder text makes sense. I'll add that.